.github/workflows/visualize-deps-of-changed-files.yml

```diff
@@ -0,0 +1,17 @@
+name: visualize dependenices of changed files in pull request
+on:
+  pull_request_target:
```

Comment:
The reason for using `pull_request_target` instead of pull_request, the PR at the forked cannot reference the GITHUB_TOKEN of the fork source.
`pull_request_target` should generally be treated with caution from a security standpoint, but the reason `pull_request_target` was originally created was to add a comment to the PR. 

> The reason to introduce the pull_request_target trigger was to enable workflows to label PRs (e.g. needs review) or to comment on the PR. The intent is to use the trigger for PRs that do not require dangerous processing, say building or running the content of the PR.

ref: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Since this purpose also does not involve hazardous processing by simply adding a comment, it would seem to make sense to use `pull_request_target` .